tests/spec/fs.symlink.spec.js

```diff
@@ -44,3 +44,33 @@ describe('fs.symlink', function() {
     });
   });
 });
+
+describe('fs.promises.symlink', function(){
+  beforeEach(function(done){
+    util.setup(function(){
+      var fsPromises = util.fs().promises;
+      fsPromises.symlink('/','/temp/mydir')
+        .then(done())
+        .catch(function(error){
+          throw error;
+        });
+    });
+  });
+  afterEach(util.cleanup);
+
+
+  it('should be a function', function() {
+    var fsPromises = util.fs().promises;
+    expect(fsPromises.symlink).to.be.a('function');
+  });
+
+
+  it('should return an error if the destination path already exists', () => {
```

Comment:
This test description doesn't match what the test is doing.  Did you want to look for an error when the dest exists?  The code below is just symlinking `/` to `/tmp`, which will work with no errors.